scripts/cli-get-changed-files.js

```diff
@@ -0,0 +1,96 @@
+const config = require('../config');
+const getChangedFilesBitbucketPipelines = require('./cli-get-changed-files-bitbucket-pipelines');
+
+const supportedCIs = ['bitbucket-pipelines'];
+const supportedAnalytics = ['snowplow'];
+
+let getChangedFilesFn;
+
+function parseCLIArgs(defaultNull) {
+    let validTestStates = {
+        inactive: 1,
+        staging: 1,
+        live: 1,
+        divert: 1
+    };
+
+    let argList = process.argv;
+    let args = {}, i, opt, thisOpt, curOpt;
+    for (i = 0; i < argList.length; i++) {
+        thisOpt = argList[i].trim();
+        opt = thisOpt.replace(/^\-+/, '');
+
+        if (opt === thisOpt) {
+            // argument value
+            if (curOpt) {
+                args[curOpt] = opt;
+            } else {
+                if (defaultNull && validTestStates[opt]) {
+                    args['state'] = opt;
+                }
+            }
+
+            curOpt = null;
+        }
+        else {
+            // argument name
+            curOpt = opt;
+            args[curOpt] = defaultNull ? null : true;
+        }
+    }
+
+    if (defaultNull) {
+        delete args.command;
+    }
+
+    return args;
+}
+
+function checkConfig() {
+    let eventsConf = config.lifecycleEvents;
+    if (!eventsConf) {
+        throw new Error(`Mojito Lifecycle events - please specify lifecycleEvents configuration in config.js.`);
+    }
+
+    let ciEnv = 'bitbucket-pipelines';
```

Comment:
@allmywant would it better to set this variable in config?

Reply:
Oh wait.. just saw the next couple of lines. Seems like this is a fallback/default.

Reply:
Will mark this one as resolved too.